src/components/UserSelector.tsx

```diff
@@ -52,14 +44,11 @@ export const UserSelector: React.FC<Props> = ({
           type="button"
           className="button"
           aria-haspopup="true"
+          aria-expanded={expanded}
```

Comment:
Consider adding keyboard navigation support to improve accessibility for the dropdown.